app/controllers/api_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class ApiController < ApplicationController
-  before_action :doorkeeper_authorize!
+  before_action :doorkeeper_authorize! unless Rails.env.development?
```

Comment:
it would be handy to feature flag this (e.g. with an env var?), so in development we can turn on/off authorisation in development